.babelrc.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
         '@babel/plugin-syntax-dynamic-import',
         ['@babel/plugin-proposal-class-properties', { loose: true }],
         'babel-plugin-macros',
-        ['emotion', { sourceMap: true, autoLabel: true }],
+        '@emotion',
```

Comment:
is this not needed here?:
```suggestion
['@emotion', { sourceMap: true, autoLabel: 'always' }]
```